src/tooling/piral-cli/src/common/http.ts

```diff
@@ -77,6 +78,14 @@ export function postFile(target: string, key: string, file: Buffer, ca?: Buffer)
           const { data, statusText, status } = error.response;
           const message = getMessage(data);
           log('unsuccessfulHttpPost_0066', statusText, status, message || '');
+        } else if (error.isAxiosError) {
+          // axios initiated error: try to parse message from error object
+          let errorMessage: string = error.errno || 'Unknown Axios Error';
+          if (error.toJSON) {
```

Comment:
I think this may be a rare case / never appearing, but potentially the more lengthy `typeof error.toJSON === 'function'` could be better suited here. What are your thoughts?

Reply:
Sounds good! Good call, apologies for the weird idioms here, pretty new to Typescript development.